packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-attach/notebook-attach.controller.js

```diff
@@ -83,4 +90,23 @@ export default class NotebookAttachController {
       true,
     );
   }
+
+  filterStorages() {
+    this.filteredStorages = this.storages
+      // Remove containers that are already on volume list
+      .filter(({ name, region }) => {
+        return !this.notebookModel.volumes
+          // eslint-disable-next-line no-shadow
```

Comment:
**question:** We tend to avoid disabling some ESLint rules. Can't be fixed here?

Reply:
fixed